.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+_Patches submitted in issues, email, or elsewhere will likely be ignored._
+
+**Descriptions**
+* Describe the changes, why they were necessary, etc
```

Comment:
You could change these to checkboxes, like:

```markdown
* [ ] Describe the changes, why they were necessary, etc
* [ ] Describe how the changes will affect existing behaviour.
* [ ] Describe how you tested and validated your changes.
* [ ] Include any relevant screenshots/evidence demonstrating that the changes work
  and have been tested.
```

Reply:
Also add one more line:

```markdown
* [ ] Add tests for the code, where appropriate.
```

Reply:
I think checkboxes is not ideal because I wanted users/developers to glance at them (i.e. review then delete everything) rather than checking off things.

Reply:
Alright, fair enough. Let's try it this way and see how it goes.

Reply:
@brndnmtthws _requested changes. /me _clicks Conversation resolved._